overlay.py

```diff
@@ -45,14 +46,21 @@ def __init__(self, bike, width=1280, height=740, bg:str=None):
 		self.data = DataFactory.create(bike_version)
 		self.device = configs["device"]
 
+		# mqtt client options
 		self.client = mqtt.Client()
 		self.client.on_connect = self._on_connect
 		self.client.on_disconnect = self.on_disconnect
 		self.client.on_log = self.on_log
+		self.client.will_set(
+			f"{str(Camera.online_root)}/{self.device}",
```

Comment:
I would extract `f"{str(Camera.online_root)}/{self.device}"` into a variable here. Probably not clear here what it does.

Reply:
Yeah, didn't know what `client.will_set` does until I googled. Would be nice to assign the publish topic to a variable that is more readable. @haotongwang

Reply:
The format string was taken from the `publish_recording_status` method, should that be changed as well? 
![image](https://user-images.githubusercontent.com/50545432/89529856-cbf3be00-d830-11ea-9d51-462df2240e85.png)

Reply:
Hm, its quite debatable, but I'd be fine with this (and similar in `publish_camera_is online`) because it follows straight from the method name. Whereas the MQTT topic in `__init__` is in a method thats generic so it won't easy to figure out straight away.